docs/usage.md

```diff
@@ -0,0 +1,153 @@
+---
```

Comment:
Hey @Florence-Njeri, **consistency is key** 🔑 😄. If you don't normally capitalize `generator` so far in these docs (as I also noted in PR [#818](https://github.com/asyncapi/generator/pull/818#discussion_r997558491)), so let's keep that consistent.

**📢 ACTION ITEM:** Please go through this entire PR and correct all your capitalized mentions of `generator` to lowercase, so that the Docs are consistent.